packages/addons/addon-depends/libconfig/package.mk

```diff
@@ -15,3 +15,8 @@ PKG_CONFIGURE_OPTS_TARGET="--enable-static \
                            --disable-shared \
                            --disable-examples \
                            --with-sysroot=$SYSROOT_PREFIX"
+
+pre_configure_target() {
+  cd ..
+  rm -rf .$TARGET_NAME
```

Comment:
This is a little unusual, what's the purpose of this?

Reply:
In tests/Makefile there is include  `-I../lib` which doesn't exist because we are building in target subfolder. Patch fixes this with absolute path.
Because package is build only for target I think this is appropriate.

Reply:
Thanks.